docs/website/docs/guides/index.md

```diff
@@ -21,11 +21,17 @@ Guides for specific frameworks:
 
 Guides for specific configurations:
 
-* [CPU](./deployment-configurations/cpu.md)
-* [CPU - Bare-Metal](./deployment-configurations/bare-metal.md)
-* [GPU - Vulkan](./deployment-configurations/gpu-vulkan.md)
-* [GPU - CUDA/ROCm](./deployment-configurations/gpu-cuda-rocm.md)
+* [:octicons-cpu-16: CPU](./deployment-configurations/cpu.md) for general
+  purpose CPU deployment
+* [:octicons-cpu-16: CPU - Bare-Metal](./deployment-configurations/bare-metal.md)
+  with minimal platform dependencies
+* [:octicons-server-16: GPU - Vulkan](./deployment-configurations/gpu-vulkan.md)
+  for cross-platform usage and interop with graphics applications
+* [:octicons-server-16: GPU - CUDA/ROCm](./deployment-configurations/gpu-cuda-rocm.md)
+  for NVIDIA/AMD-specific optimizations
```

Comment:
Not super thrilled with this phrasing "-specific optimizations"... any suggestions for other reasons that users would target CUDA/ROCm over Vulkan? "-specific _features_" maybe?

Reply:
-specific solutions?